dex-it-common/src/main/scala/com/wavesplatform/dex/it/docker/PortBindingKeeper.scala

```diff
@@ -0,0 +1,81 @@
+package com.wavesplatform.dex.it.docker
+
+import cats.implicits._
+import com.github.dockerjava.api.command.CreateContainerCmd
+import com.github.dockerjava.api.model.Ports.Binding
+import com.github.dockerjava.api.model.{ExposedPort, PortBinding}
+import com.wavesplatform.dex.domain.utils.ScorexLogging
+
+import java.net.ServerSocket
+import java.util
+import scala.jdk.CollectionConverters._
+import scala.util.{Random, Using}
+
+object PortBindingKeeper extends ScorexLogging {
+
+  case class AvailablePortsNotFound() extends Throwable
+
+  private val stringRange = Option(System.getenv("TEST_PORT_RANGE"))
+    .getOrElse(throw new RuntimeException("Please specify the TEST_PORT_RANGE environment variable"))
+
+  private val portsRange = parsePortRange()
```

Comment:
We don't use `stringRange` later. Probably it'd better to write something like:
```scala
private val portsRange = Option(System.getenv("TEST_PORT_RANGE"))
    .map(parsePortRange)
    .getOrElse(throw new RuntimeException("Please specify the TEST_PORT_RANGE environment variable"))
```
instead.